### PR TITLE
release: v1.0.0a7

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis ioannis.tsanaktsidis@cern.ch

### CODE REVIEW

File: invenio_deposit/ext.py [hunk 1]
```diff
@@ -134,8 +134,7 @@ def init_app(self, app):
 
         @blueprint.record_once
         def extend_default_endpoint_prefixes(_):
-            """TODO."""
-            # FIXME remove hasattr() after 1.0.0a7 is released
```
Comment (lnielsen) [inline invenio_deposit/ext.py]: I think the FIXME is still relevant.

Comment (ioannistsanaktsidis) [inline invenio_deposit/ext.py]: @lnielsen  Yes just saw it, I guess the if statement should be removed completely after the merge ?

Comment (jirikuncar) [inline invenio_deposit/ext.py]: @ioannistsanaktsidis before the release we should address #138 in separate PR.

Comment (ioannistsanaktsidis) [inline invenio_deposit/ext.py]: @lnielsen  @jirikuncar Thnx! Ok then I leave this PR as it is so as  to contain onyl the release.

**Comment by lnielsen** [comment]
LGTM

**Pull request merged**